Rhino tutorial (#120)
Appsilon/shiny.router: vignettes/rhino.Rmd

---
title: "Tutorial: shiny.router in Rhino applications"
output: rmarkdown::html_vignette
vignette: >
  %\VignetteIndexEntry{Tutorial: shiny.router in Rhino applications}
  %\VignetteEngine{knitr::knitr}
  %\VignetteEncoding{UTF-8}
---

# Introduction

`rhino` is a `shiny` framework focused on building applications with
the best software engineering practices in mind. This tutorial will show you
how you can use `shiny.router` in a Rhino app.

You can read more about `rhino` on the [documentation page](https://appsilon.github.io/rhino/).

# Setup Rhino application

The first step is to create a Rhino application.
Before that, make sure that you have `rhino` installed:

```r
install.packages("rhino")
```

Now, initialize a new Rhino application. You can do it either by using RStudio Wizard
or a function `rhino::init("rhino_router")`.
More details on how to create a Rhino app can be found in the [Rhino tutorial](https://appsilon.github.io/rhino/articles/tutorial/create-your-first-rhino-app.html#create-an-initial-application).

# Add the basic content

> To speed things up, we will use a basic version of the Rhino application that
> you can build by following the [Rhino tutorial](https://appsilon.github.io/rhino/articles/tutorial/create-your-first-rhino-app.html).

Reviewer: Suggestion:

At first I understood that I need to follow the linked Rhino tutorial before continuing to the next sections 🙈 

Maybe we can rephrase that to

```suggestion
To speed things up, we will use a basic version of the Rhino application described in the [Rhino tutorial](https://appsilon.github.io/rhino/articles/tutorial/create-your-first-rhino-app.html).
```
